modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -37,7 +37,13 @@ The {ai-full} supports installing {product-title} on premises in a connected env
 * Highly available {product-title} or {sno} (SNO)
 
 * {product-title} on bare metal, Nutanix, or vSphere with full platform integration, or other virtualization platforms without integration
-* Optionally {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage)
+
+* Optionally, {VirtProductName}, multicluster engine, {lvms-first}, and {rh-storage}
```

Comment:
```suggestion
* Optional: {VirtProductName}, multicluster engine, {lvms-first}, and {rh-storage}
```
I think this is the standard way we refer to optional items in a numbered or bulleted list.